common/capture.go

```diff
@@ -48,7 +48,7 @@ var (
 )
 
 func initCaptureTypes() {
-	CaptureTypes["ovsbridge"] = CaptureType{Allowed: []string{"ovssflow", "pcapsocket"}, Default: "ovssflow"}
+	CaptureTypes["ovsbridge"] = CaptureType{Allowed: []string{"ovssflow (Flow Samples)", "ovssflow (Counter Samples)", "ovssflow (Flow + Counter Samples)", "pcapsocket"}, Default: "ovssflow (Flow Samples)"}
```

Comment:
I would prefer to keep only one probe but adding options to the API(capture) that will be passed/used when registering the probe (see https://github.com/skydive-project/skydive/blob/master/flow/probes/ovssflow.go#L241)

I think playing with sampling/polling parameter should be enough to have the expected behavior without modifying the sflow agent code ?
https://github.com/skydive-project/skydive/blob/master/flow/probes/ovssflow.go#L190